os/os.module.go

```diff
@@ -0,0 +1,224 @@
+package os
+
+// Copyright 2022 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"reflect"
+	"strings"
+
+	"github.com/go-python/gpython/py"
+)
+
+func init() {
+
+	methods := []*py.Method{
+		py.MustNewMethod("getcwd", getCwd, 0, "Get the current working directory"),
+		py.MustNewMethod("chdir", chdir, 0, "Change the current working directory"),
+		py.MustNewMethod("getenv", getenv, 0, "Return the value of the environment variable key if it exists, or default if it doesn’t. key, default and the result are str."),
+		py.MustNewMethod("getpid", getpid, 0, "Return the current process id."),
+		py.MustNewMethod("putenv", putenv, 0, "Set the environment variable named key to the string value."),
+		py.MustNewMethod("unsetenv", unsetenv, 0, "Unset (delete) the environment variable named key."),
+		py.MustNewMethod("_exit", _exit, 0, "Immediate program termination."),
+		py.MustNewMethod("system", system, 0, "Run shell commands, prints stdout directly to deault"),
+	}
+
+	globals := py.StringDict{
+		"error":   py.OSError,
+		"environ": getEnvVariables(),
+	}
+
+	py.RegisterModule(&py.ModuleImpl{
+		Info: py.ModuleInfo{
+			Name: "os",
+			Doc:  "Miscellaneous operating system interfaces",
+		},
+		Methods: methods,
+		Globals: globals,
+	})
+}
+
+// gets and parses all process enviroment variables
+func getEnvVariables() py.StringDict {
+	env_variables := os.Environ() // this gets all the enviroment variables
+	dict := py.NewStringDict()
+	for _, evar := range env_variables {
+		key_value := strings.Split(evar, "=") // returns a []string containing [key,value]
```

Comment:
Here you should probably use `strings.SplitN(evar, "=", 2)` in case there are values containing `=`